src/main/java/com/mock/preparation/ds/EvenOddArray.java

```diff
@@ -0,0 +1,28 @@
+package mock.preparation.ds;
+
+import java.util.Scanner;
+
+public class EvenOddArray {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        scanner.nextInt();
+        try {
+            int[] a = {5, 6, 7, 8, 9, 10};
+            for (int i = 0; i <= a.length; i++) {
+                for (int j = 0; j <= a.length; j++) {
+                    if (a[i] % 2 == 0) {
+                        System.out.println("It is an even number\n");
```

Comment:
It will be good if you can print the number or index of number. it does not make sense only saying its odd or even number. Which number we talking about?